### ISSUE TITLE
IndexError: list index out of range + CIRCexplorer2 parse

### ISSUE THREAD

**Issue by BirongZhang** [YangLab/CIRCexplorer2#58]
Hi all,

Thanks for developing such a useful tool!

I used STAR/2.7.0e and Homo_sapiens.GRCh38.103 reference to get the Chimeric.out.junction file. But in the next steps_Parsing for Circular RNA Fusion Junction Reads, I ran into some problems:

Here is my command and error:
<img width="904" alt="Screenshot 2021-10-08 at 13 30 04" src="https://user-images.githubusercontent.com/74430395/136557037-23a7ebdc-fa27-4363-9c69-eae4e6196527.png">

Here is my CIRCexplorer2_parse.log:
<img width="645" alt="Screenshot 2021-10-08 at 13 30 31" src="https://user-images.githubusercontent.com/74430395/136557109-0c6557ea-7731-434f-a553-e0350280e6f0.png">

Here is my Chimeric.out.junction file:
<img width="1039" alt="Screenshot 2021-10-08 at 13 31 10" src="https://user-images.githubusercontent.com/74430395/136557186-303184d6-abb5-4b0c-bb28-aa5faa0b4246.png">

Any advice would be appreciated. Thanks.

**Comment by BirongZhang** [comment]
Hi all, 
I solved my problem.

Before that, I used the "-- chimOutJunctionFormat 1" command in STAR, so I got the chimeric.out.junction file with some extra lines at the end, which looks something like this：
<img width="833" alt="Screenshot 2021-10-08 at 15 00 44" src="https://user-images.githubusercontent.com/74430395/136570313-f6560d57-28d4-410f-8216-0f78512884e1.png">

I then manually removed them and ran the Parse command to get the above error. 
Just use "-- chimOutJunctionFormat 0" or by default, problem solved.
